### ISSUE TITLE
Uncaught TypeError: Cannot read property 'severity' of undefined

### ISSUE THREAD

**Issue by avdv** [mulesoft/api-workbench#334]
[Enter steps to reproduce below:]

1. open a RAML 1.0 project or switch to a .raml file

**Atom Version**: 1.11.2
**Electron Version**: 0.37.8
**System**: linux 4.7.6-1-ARCH
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package, v0.8.39

### Stack Trace

Uncaught TypeError: Cannot read property 'severity' of undefined

```
At /home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/typesystem.js:87

TypeError: Cannot read property 'severity' of undefined
    at Status.addSubStatus (/home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/typesystem.js:87:31)
    at /home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/typesystem.js:1046:74
    at Array.forEach (native)
    at InheritedType.AbstractType.validateDirect (/home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/typesystem.js:1046:37)
    at InheritedType.AbstractType.validate (/home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/typesystem.js:1089:34)
    at PropertyIs.MatchesProperty.validateProp (/home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/restrictions.js:54:20)
    at PropertyIs.check (/home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/restrictions.js:294:31)
    at /home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/typesystem.js:1046:89
    at Array.forEach (native)
    at InheritedType.AbstractType.validateDirect (/home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/typesystem.js:1046:37)
    at Example.validateValue (/home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/metainfo.js:247:37)
    at Example.validateSelf (/home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/metainfo.js:219:34)
    at /home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/typesystem.js:685:15
    at Array.forEach (native)
    at InheritedType.AbstractType.validateMeta (/home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/typesystem.js:684:29)
    at InheritedType.AbstractType.validateType (/home/claudio/.atom/packages/api-workbench/node_modules/raml-typesystem/dist/src/typesystem.js:569:18)
    at TypeDeclarationValidator.validate (/home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/ast.core/linter.js:2022:34)
    at Object.validate (/home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/ast.core/linter.js:658:44)
    at ASTNodeImpl.validate (/home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/highLevelImpl.js:940:16)
    at /home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/ast.core/linter.js:489:15
    at Array.forEach (native)
    at validateBasic (/home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/ast.core/linter.js:484:12)
    at Object.validate (/home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/ast.core/linter.js:686:13)
    at ASTNodeImpl.validate (/home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/highLevelImpl.js:940:16)
    at Object.validate (/home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/ast.core/linter.js:595:36)
    at ASTNodeImpl.validate (/home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/highLevelImpl.js:940:16)
    at /home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/ast.core/linter.js:489:15
    at Array.forEach (native)
    at validateBasic (/home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/ast.core/linter.js:484:12)
    at Object.validate (/home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/ast.core/linter.js:686:13)
    at ASTNodeImpl.validate (/home/claudio/.atom/packages/api-workbench/node_modules/raml-1-parser/dist/raml1/highLevelImpl.js:940:16)
    at c (/home/claudio/.atom/packages/api-workbench/main.js:2:5537)
    at u (/home/claudio/.atom/packages/api-workbench/main.js:2:5259)
    at p (/home/claudio/.atom/packages/api-workbench/main.js:2:4583)
    at S (/home/claudio/.atom/packages/api-workbench/main.js:2:6045)
    at Workspace.module.exports.Workspace.observeTextEditors (/usr/share/atom/resources/app.asar/src/workspace.js:330:9)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "metrics",
      "linter-raml"
    ],
    "telemetryConsent": "no"
  },
  "api-workbench": {}
}
```

### Installed Packages

```coffee
# User
api-workbench, v0.8.39 (inactive)
atom-handlebars, v1.3.0 (inactive)
autocomplete-plus, v2.33.1 (inactive)
editorconfig, v1.5.3 (inactive)
ember-cli-helper, v0.9.0 (inactive)
ember-snippets, v1.4.0 (inactive)
emmet, v2.4.3 (inactive)
git-plus, v5.18.0 (inactive)
git-time-machine, v1.5.3 (inactive)
jsonpp, v0.1.1 (inactive)
linter, v1.11.18 (inactive)
linter-raml, v1.0.0 (inactive)
minimap, v4.25.5 (inactive)
pretty-json, v1.6.1 (inactive)
quick-move-file, v0.7.0 (inactive)
raml, v0.2.0 (inactive)
underline-trailing-whitespace, v1.4.0 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.2.0 (inactive)
base16-tomorrow-light-theme, v1.2.0 (inactive)
one-dark-ui, v1.6.0 (active)
one-light-ui, v1.6.0 (inactive)
one-dark-syntax, v1.3.0 (active)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.7.0 (inactive)
archive-view, v0.61.1 (inactive)
autocomplete-atom-api, v0.10.0 (inactive)
autocomplete-css, v0.11.2 (inactive)
autocomplete-html, v0.7.2 (inactive)
autocomplete-plus, v2.33.1 (inactive)
autocomplete-snippets, v1.11.0 (inactive)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (inactive)
background-tips, v0.26.1 (inactive)
bookmarks, v0.42.0 (inactive)
bracket-matcher, v0.82.1 (inactive)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (inactive)
dev-live-reload, v0.47.0 (inactive)
encoding-selector, v0.22.0 (inactive)
exception-reporting, v0.40.0 (inactive)
find-and-replace, v0.201.1 (inactive)
fuzzy-finder, v1.4.0 (inactive)
git-diff, v1.1.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (inactive)
image-view, v0.59.0 (inactive)
incompatible-packages, v0.26.1 (inactive)
keybinding-resolver, v0.35.0 (inactive)
line-ending-selector, v0.5.0 (inactive)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (inactive)
metrics, v1.0.0 (inactive)
notifications, v0.65.1 (inactive)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2-hotfix1 (inactive)
snippets, v1.0.2 (inactive)
spell-check, v0.68.2 (inactive)
status-bar, v1.4.1 (inactive)
styleguide, v0.47.0 (inactive)
symbols-view, v0.113.1 (inactive)
tabs, v0.101.0 (inactive)
timecop, v0.33.2 (inactive)
tree-view, v0.209.3 (inactive)
update-package-dependencies, v0.10.0 (inactive)
welcome, v0.35.1 (inactive)
whitespace, v0.33.0 (inactive)
wrap-guide, v0.38.2 (inactive)
language-c, v0.52.1 (inactive)
language-clojure, v0.21.0 (inactive)
language-coffee-script, v0.47.2 (inactive)
language-csharp, v0.12.1 (inactive)
language-css, v0.37.1 (inactive)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (inactive)
language-go, v0.42.1 (inactive)
language-html, v0.45.1 (inactive)
language-hyperlink, v0.16.0 (inactive)
language-java, v0.23.0 (inactive)
language-javascript, v0.119.0 (inactive)
language-json, v0.18.2 (inactive)
language-less, v0.29.5 (inactive)
language-make, v0.22.2 (inactive)
language-mustache, v0.13.0 (inactive)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.35.0 (inactive)
language-php, v0.37.2 (inactive)
language-property-list, v0.8.0 (inactive)
language-python, v0.45.0 (inactive)
language-ruby, v0.69.0 (inactive)
language-ruby-on-rails, v0.25.0 (inactive)
language-sass, v0.56.0 (inactive)
language-shellscript, v0.22.4 (inactive)
language-source, v0.9.0 (inactive)
language-sql, v0.23.0 (inactive)
language-text, v0.7.1 (inactive)
language-todo, v0.28.0 (inactive)
language-toml, v0.18.0 (inactive)
language-xml, v0.34.9 (inactive)
language-yaml, v0.26.0 (inactive)

# Dev
No dev packages
```

**Comment by KonstantinSviridov** [comment]
Hi, @avdv 

Thanks for the report. The fix will be released in 0.8.40

Regards, Konstantin

**Comment by ddenisenko** [comment]
Hopefully, our reproduction sequence matches the original issue as the case we reproduced was pretty rare one.
